.github/workflows/ci.yml

```
@@ -0,0 +1,309 @@
+        env:
+          DATABASE_URL: postgresql://postgres:postgres@localhost:5432/test_db
+
+      - name: Generate coverage report
+        run: cargo tarpaulin --all-targets --all-features --out Xml --output-dir coverage --verbose
+        env:
+          DATABASE_URL: postgresql://postgres:postgres@localhost:5432/test_db
+          RUST_LOG: debug
+
+      - name: Generate coverage summary
+        id: coverage
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:3:32: Double quote to prevent globbing and word splitting

(shellcheck)